notebooks/notebooks/paper/katago-new-ckpt-evals.ipynb

```diff
@@ -0,0 +1,199 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.                        label=r"\texttt{Latest}",
</pre><p>nit: maybe label should include that the number of visits is 200 here, though I guess the win rate turns out to be the same for all visit counts at this number of steps</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Added 200 visits.